infrastructure/kube/keep-test/tbtc-v2-monitoring/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+bases:
+  - ../../templates/tbtc-v2-monitoring
+
+images:
+  - name: tbtc-v2-monitoring
+    newName: gcr.io/keep-test-f3e0/tbtc-v2-monitoring
+    newTag: latest
+
+configMapGenerator:
+  - name: tbtc-v2-monitoring-config
+    literals:
+      - environment=testnet
+      - large-deposit-threshold-sat=10000000 # 0.1 BTC for testing purposes
+
+secretGenerator:
+  - name: tbtc-v2-monitoring-config
+    envs:
+      - .env.secret
```

Comment:
This will set secrets from `.env.secret` in the configmap when executing `kubectl apply -k ./`? 🤔

Reply:
Yeah, that will read `.env.secret` and create a k8s secret object with the same keys and values as in the file.